www/base/src/components/MultipleWorkersActionsModal/MultipleWorkersActionsModal.tsx

```diff
@@ -36,6 +37,42 @@ const workerToSelectOption = (worker: Worker): SelectOption => {
   return { value: worker, label: worker.name }
 }
 
+const Control = (props: any) => {
```

Comment:
Control should be a more explicit name, it's hard to figure out what exactly `Control` refers to.